index.ftd

```diff
@@ -1706,41 +1752,604 @@ width: fill-container
-    - Label 4: index.html
-    - Label 5: index.html
-- Section 5: basic-footer/
-  - Sub-section 5:
-    - Label 1: index.html
-    - Label 2: index.html
-    - Label 3: index.html
-    - Label 4: index.html
-    - Label 5: index.html
\ No newline at end of file
+-- component post-card-ui:
+caption title:
+string timestamp:
+ftd.image-src image: 
+string url:
+body body:
+optional common.author-meta author:
+common.tags-list list tags:
+
+-- ftd.column:
+width: fill-container
+
+-- ftd.row:
+width: fill-container
+spacing.fixed.px: 60
+
+-- ftd.column:
+
+-- ftd.row:
+width: fill-container
+
+-- topic: $obj.title
+$loop$: $post-card-ui.tags as $obj
+tag-link: $obj.tag-link
+active: true
+
+-- ftd.text: $post-card-ui.timestamp
+color: $inherited.colors.text
+role: $inherited.types.label-small
+padding-horizontal.px: 12
+padding-vertical.px: 5
+
+-- end: ftd.row
+
+-- ftd.text: $post-card-ui.title
+color: $inherited.colors.text
+role: $inherited.types.heading-small
+margin-top.px: 24
+
+-- ftd.text:
+color: $inherited.colors.text
+role: $inherited.types.copy-regular
+margin-top.px: 12
+
+$post-card-ui.body
+
+-- end: ftd.column
+
+-- ftd.image:
+src: $post-card-ui.image
+height.fixed.px: 200
+width.fixed.px: 300
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: post-card-ui
+
+
+
+
+
+
+
+
+
+
+-- component featured-post-card:
+caption title:
+optional body body:
+optional string url:
+optional string timestamp:
+optional ftd.image-src image:
+common.author-meta author:
+optional string extra-text:
+optional string cta-primary: 
+optional string cta-primary-url:
+optional string cta-secondary: 
+optional string cta-secondary-url:
+
+-- ftd.column:
+width: fill-container
+padding-vertical.px: 120
+
+-- ftd.row:
+width: fill-container
+border-left-width.px: 19
+border-color: $inherited.colors.accent.primary
+spacing: space-between
+
+-- ftd.column:
+width.fixed.px: 445
+margin-left.px: 24
+align-self: center
+
+-- ftd.text: $featured-post-card.title
+color: $inherited.colors.text
+role: $inherited.types.heading-large
+
+-- ftd.text:
+color: $inherited.colors.text
+role: $inherited.types.copy-regular
+margin-top.px: 24
+
+$featured-post-card.body
+
+-- ftd.row:
+padding-horizontal.px: 20
+padding-vertical.px: 12
+spacing.fixed.px: 8
+background.solid: $inherited.colors.cta-primary.base
+align-content: center
+margin-top.px: 24
+
+-- ftd.text: Button
+color: $inherited.colors.cta-primary.text
+role: $inherited.types.copy-regular
+link: /
+
+-- ftd.image:
+src: $assets.files.assets.tick.svg
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- ftd.image:
+width.fixed.percent: 55
+src: $featured-post-card.image
+align-self: end
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: featured-post-card
+
+
+
+
+
+
+
+
+
+
+-- component featured-post:
+common.post-meta post-data:
+optional string cta-primary: 
+optional string cta-primary-url:
+optional string cta-secondary: 
+optional string cta-secondary-url:
+optional string extra-text:
```

Comment:
```suggestion
optional body body:
```

Reply:
body is the property which is coming from post-data

Reply:
@saurabh-lohiya this is not the correct format of passing `body`
```ftd
-- featured-post-card: $featured-post.post-data.title
timestamp: $featured-post.post-data.published-on
author: $featured-post.post-data.author
image: $featured-post.post-data.post-image
extra-text: $featured-post.extra-text
cta-primary: $featured-post.cta-primary
cta-primary-url: $featured-post.cta-primary-url
cta-secondary: $featured-post.cta-secondary
cta-secondary-url: $featured-post.cta-secondary-url

body: $featured-post.post-data.body
```
The correct format is:
```ftd
-- featured-post-card: $featured-post.post-data.title
timestamp: $featured-post.post-data.published-on
author: $featured-post.post-data.author
image: $featured-post.post-data.post-image
extra-text: $featured-post.extra-text
cta-primary: $featured-post.cta-primary
cta-primary-url: $featured-post.cta-primary-url
cta-secondary: $featured-post.cta-secondary
cta-secondary-url: $featured-post.cta-secondary-url

$featured-post.post-data.body
```